tests/PropertyList.py

```diff
@@ -102,14 +102,14 @@ def testExists(self):
 
     def testGetVector(self):
         apl = dafBase.PropertyList()
-        v = (42, 2008, 1)
+        v = [42, 2008, 1]
         apl.setInt("ints", v)
-        apl.setInt("ints2", (10, 9, 8))
+        apl.setInt("ints2", [10, 9, 8])
```

Comment:
I do think we need to be able accept any sequence type (longer term than this PR, but probably before any pybind11 changes hit master).  I'm not particularly bothered by return types, and I think we should catalog our use cases before adding views; I think in a lot of cases we can just switch to using ndarray in C++ and have natural conversion to Numpy instead of dealing with `std::vector` vs `list` or `tuple`.